### PR TITLE
Being able to add a character to a question

### PR DESCRIPTION
**Pull request by timkrief**
Peek 2021-04-09 19-32
I tested with an old dialog and it loads as no character as expected, but I don't know if there are other edgecases to test

### CODE REVIEW

**Comment by timkrief** [comment]
Need some more testing... I think it does not save/load character name properly

**Review by Jowan-Spooner — COMMENTED** [review]
Looking very promising!

File: addons/dialogic/Editor/Pieces/Question.gd [hunk 1]
```diff
@@ -8,14 +8,54 @@ var editorPopup
...
 
+onready var portrait_picker = $PanelContainer/VBoxContainer/Header/PortraitPicker
+
+func _ready():
+	$PanelContainer/VBoxContainer/Header/CharacterPicker.connect('character_selected', self , '_on_character_selected')
+	portrait_picker.get_popup().connect("index_pressed", self, '_on_portrait_selected')
+
+	var c_list = DialogicUtil.get_sorted_character_list()
+	if c_list.size() == 0:
+		$PanelContainer/VBoxContainer/Header/CharacterPicker.visible = false
```
[the remark this answers is not in the document]
Comment (timkrief) [reply addons/dialogic/Editor/Pieces/Question.gd:24]: I tried to stay consistent with the way it's done in textblock :sweat_smile:

**Comment by timkrief** [comment]
I think now it supports loading of old data, I can't find issue anymore so I'm reopening this

**Pull request merged**